src/Symfony/Bundle/FrameworkBundle/Tests/Functional/Bundle/TestBundle/AutowiringTypes/AnnotationReader.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Functional\Bundle\TestBundle\AutowiringTypes;
+
+use Doctrine\Common\Annotations\Reader;
+
+class AnnotationReader
```

Comment:
@dunglas I've renamed it to `AutowiredServices`. One can add an attribute to the constructor for testing another service and test it in the `AutowiringTypesTest`. Foo seemed too generic in this case.